src/Workspaces/Core/Desktop/Workspaces.Desktop.csproj

```diff
@@ -109,9 +108,29 @@
     <Compile Include="Workspace\MSBuild\MSBuildProjectLoader.cs" />
     <Compile Include="Workspace\SolutionSize\SolutionSizeTracker.cs" />
     <Compile Include="Workspace\Esent\EsentPersistentStorageService.cs" />
+    <Compile Include="Workspace\SQLite\Interop\OpenFlags.cs" />
```

Comment:
It doesn't have to block this checkin, but since AFAIK sqllite is portable, we should consider moving this into Workspaces directly so that hosts on other platforms can use it.

Not sure if that's necessary for @mhutch for Monodevelop or not though.